ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerSession.scala

```diff
@@ -33,8 +38,11 @@ private[testtool] final class LedgerSession(
       identifierSuffix: String,
   ): Future[LedgerTestContext] =
     participantSessions.flatMap { sessions =>
+      val sessions2 =
+        if (config.shuffleParticipants) scala.util.Random.shuffle(sessions)
```

Comment:
This makes sense, but in which context would it be a _bad_ idea?

Reply:
When you have two participants that are configured in a specific way and you want tests to always it the two participants in the same order (within the context of a single test).

Reply:
That sort of makes sense, except we use a ring buffer, so adding a test _before_ that test case will change the order. In addition, we don't always run tests in the same order, so the participant order might be different from run to run.

Reply:
The participant sessions are initialized on construction based on the ring buffer, so every test case will always pick either the first or the first two participants passed to the application (because we don't have any test case that picks up more than two participants).

Reply:
My comment refers to the behavior _without_ this newly introduced flag, which allows to shuffle participants against which the test is ran.